src/app/components/chart/areaStackedTimeSeries.chart.directive.js

```diff
@@ -29,11 +29,18 @@
             scope.legend = true;
 
             var chart;
+            
+            $rootScope.$on("angular-resizable.resizeEnd", function (event, args) {
+                var element = document.getElementById(args.id);
+                var child = element.childNodes[3].childNodes[0].childNodes[0].childNodes[3];
+                scope.updatedID = child.id;
```

Comment:
Might be better to use a `dirty` flag in the element instead of updatedID in the scope.